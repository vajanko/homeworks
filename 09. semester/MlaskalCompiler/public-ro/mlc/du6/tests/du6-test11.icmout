ERROR: Requested bad value type on IP=20
