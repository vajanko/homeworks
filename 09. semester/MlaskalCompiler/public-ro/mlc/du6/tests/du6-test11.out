Error (line 12): Expression may not be passed by reference
There were some errors 1 and warnings 0
